commands/misc/credits.js

```diff
@@ -12,17 +12,15 @@ module.exports = class CreditsCommand extends commando.Command {
                 Displays bot Developer(s) and such.
             `,
             examples: ['credits'],
-        })
+        });
     }
 
     async run(msg) {
-        if (msg.author.bot) return;
```

Comment:
dont really need this because default discord.js commando auto ignore bot messages